parts/k8s/containeraddons/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -110,12 +110,12 @@ spec:
           imagePullPolicy: IfNotPresent
           name: heapster
           resources:
-            limits:
-              cpu: 88m
-              memory: 204Mi
             requests:
-              cpu: 88m
-              memory: 204Mi
+              cpu: {{ContainerCPUReqs "heapster"}}
```

Comment:
Moving these resource configuration values to a more maintainable/testable place.